paddle/fluid/pybind/pir.cc

```diff
@@ -669,10 +673,15 @@ void BindBlock(py::module *m) {
              }
              return param_list;
            })
-      .def("refresh_stopgradient", [](Block &self) {
-        for (auto &op : self) {
-          RefreshOpStopgradients(&op);
-        }
+      .def("refresh_stopgradient",
```

Comment:
```suggestion
      .def("refresh_stop_gradient",
```
在Python段我们统一使用的是stop_gradient